src/controllers/tileController.ts

```diff
@@ -1,28 +1,49 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
 import { AppLocals } from "../types/app";
+import asyncControllerHandler from "../errors/asyncControllerHandler";
+import notFound from "../errors/notFound";
+import { GroutError } from "../errors/groutError";
+import { ErrorType } from "../errors/errorType";
+
+const parseIntParam = (param: string): number => {
+    // Native parseInt is not strict (ignores whitespace and trailing chars) so test with a regex
+    if (!/^\d+$/.test(param)) {
+        throw new GroutError(
+            `"${param}" is not an integer`,
+            ErrorType.BAD_REQUEST
+        );
+    }
+    return parseInt(param, 10);
```

Comment:
Why not use quinary? Decimal is so 2024.

Reply:
Banking on retro vibes being in next season.